playground/flags/evaluation-tags-example.html

```diff
@@ -0,0 +1,536 @@
+        function log(message, type = 'info') {
+            const logContainer = document.getElementById('logContainer');
+            const entry = document.createElement('div');
+            entry.className = `log-entry ${type}`;
+            const timestamp = new Date().toLocaleTimeString();
+            entry.textContent = `[${timestamp}] ${message}`;
+            logContainer.appendChild(entry);
+            logContainer.scrollTop = logContainer.scrollHeight;
+        }
+
+        function updateStatus(message, type = 'success') {
+            const statusDiv = document.getElementById('status');
+            const statusText = document.getElementById('statusText');
+            
+            statusDiv.className = 'status';
+            if (type === 'error') {
+                statusDiv.classList.add('error');
+            } else if (type === 'warning') {
+                statusDiv.classList.add('warning');
+            }
+            
+            statusText.textContent = message;
+        }
+
+        function clearLogs() {
+            const logContainer = document.getElementById('logContainer');
+            logContainer.innerHTML = '<div class="log-entry info">Logs cleared</div>';
+        }
+
+        function initializePostHog() {
+            const apiKey = document.getElementById('apiKey').value;
+            const apiHost = document.getElementById('apiHost').value;
+            const personalKey = document.getElementById('personalKey').value;
+            const distinctId = document.getElementById('distinctId').value;
+            const debugMode = document.getElementById('debugMode').value === 'true';
+
+            if (!apiKey) {
+                log('Error: API key is required', 'error');
+                updateStatus('Missing API key', 'error');
+                return;
+            }
+
+            try {
+                // Configuration object with evaluation environments
+                const config = {
+                    api_host: apiHost,
+                    debug: debugMode,
+                    loaded: function(posthog) {
+                        log('PostHog loaded successfully', 'success');
+                        updateStatus('PostHog initialized and ready', 'success');
+                    },
+                    // Add evaluation environments if tags are present
+                    ...(evaluationTags.length > 0 && {
+                        evaluation_environments: evaluationTags
+                    })
+                };
+
+                // Add personal API key if provided
+                if (personalKey) {
+                    config.personal_api_key = personalKey;
+                }
+
+                log('Initializing PostHog with config:', 'info');
+                log(JSON.stringify(config, null, 2), 'info');
+
+                // Initialize PostHog
+                if (window.posthog) {
+                    posthogInstance = window.posthog;
+                    posthogInstance.init(apiKey, config);
```

Comment:
`window.posthog` will change once the client is loaded from the snippet. `posthogInstance` won't have access to data like feature flags once they're loaded. Moving this to the `loaded` callback in the config should fix this.